composer.json

```
@@ -15,17 +15,17 @@
     },
     "require-dev" : {
         "squizlabs/php_codesniffer" : "2.9.2",
-        "phpunit/phpunit" : "6.5.*",
+        "phpunit/phpunit" : "^8",
         "facebook/webdriver" : "dev-master",
         "phpmd/phpmd": "^2.6",
-        "phan/phan": ">1.2.2"
+        "phan/phan": ">2.2.4"
     },
     "scripts": {
       "pre-install-cmd": "mkdir -p project/libraries"
     },
     "autoload" : {
         "psr-4": {
-            "LORIS\\": "src/"
+            "LORIS\\":  "src/"
```

extra space, not show blocker
```suggestion
            "LORIS\\": "src/"
```